openeo_processes_dask/core.py

```diff
@@ -67,12 +58,79 @@ def wrapper(*args, parameters: Optional[dict[str]] = None, **kwargs):
     return wrapper
 
 
-process_registry = {
-    func.__name__.strip("_"): process(func) for func in standard_processes
-}
+class ProcessRegistry(MutableMapping):
+    """
+    The process registry is basically a dictionary mapping from process_id to the callable implementation.
+    It also allows registering aliases for process_ids.
+    """
+
+    def __init__(self, *args, **kwargs):
+        self.store = dict()  # type: dict[str, Callable]
+        self.aliases = dict()  # type: dict[str, str]
+
+        self.update(dict(*args, **kwargs))  # use the free update to set keys
+
+    def __getitem__(self, key) -> Callable:
+        return self.store[self._keytransform(key)]
```

Comment:
If I enter the name of an alias here, it would KeyError? If we are allowing aliases, would be cool to support them here?

Reply:
nice!